src/Sulu/Component/Webspace/Manager/WebspaceManager.php

```diff
@@ -62,6 +62,16 @@ public function findWebspaceByKey($key)
     }
 
     /**
+     * Return true if the webspace with the given key exists.
```

Comment:
I think we should also rename `findWebspaceByKey` to `getWebspace`. But this would probably be a (bigger) BC break.